BinaryEncoding.md

```diff
@@ -119,11 +119,15 @@ The type section declares all function signatures that will be used in the modul
 | entries | `type_entry*` | repeated type entries as described below |
 
 #### Type entry
-| Field | Type | Description |
+| Field | Type/Value | Description |
 | ----- |  ----- | ----- |
+| constructor | `0x05` | the function type constructor |
```

Comment:
And now I remember the original question that motivated my post: given this single index space, then why does the `constructor` field need to start at any other value than `0`?  Function (and later struct, array) constructors are not in the same index space anymore.